devmaster/themes/boots/template.php

```diff
@@ -808,6 +808,11 @@ function boots_preprocess_page(&$vars){
   // Render stuff
   $vars['tabs_rendered'] = render($vars['tabs']);
   $vars['sidebar_first_rendered'] = render($vars['page']['sidebar_first']);
+
+  // Indicate the project is inactive.
+  if ((int) $vars['node']->status == NODE_NOT_PUBLISHED) {
+    $vars['subtitle'] .= ' <small>' . t('Inactive') .  '</small>';
```

Comment:
This is good in the header, but there is still a "Status Disabled" link in the info bar.